lib/transmit.js

```diff
@@ -284,9 +284,7 @@ internals.end = function (env, event, err) {
     env.team = null;
 
     if (request.raw.res.writableEnded) {
-        if (!event) {
-            request.info.responded = Date.now();
-        }
+        request.info.responded = Date.now();
```

Comment:
Why did you remove the `!event` check?

Reply:
Because the early `req.destroy()` call removes the only test case that has a non-`null` event, making it fail coverage. I don't think there are any other ways to end up with `res.writableEnded` along with an event, and if there is such a corner case, the damage will be negligible.

Reply:
Alright, thanks for the explanation.